stevepetebruce/BookMi.Place-Landing
# Post email addresses to MailChimp

#### What's this PR do?
Changes the email address collection form to point to MailChimp

##### Background context
I started work on incorporating my email capture API app but then learned that MailChimp can handle all of this for us.

#### Where should the reviewer start?
index.html

#### How should this be manually tested?
Try adding an email address.. 
Check the console - that there's no errors
Check MailChimp list - that they've been added

#### Screenshots
![image](https://user-images.githubusercontent.com/1453680/36945673-929f5778-1fa9-11e8-8c72-103fefd9c811.png)

![image](https://user-images.githubusercontent.com/1453680/36945676-96bca3e2-1fa9-11e8-85bc-1516c0d937bb.png)


![image](https://user-images.githubusercontent.com/1453680/36945681-9af51796-1fa9-11e8-9df6-89271fa37e13.png)

---
